.travis.yml

```diff
@@ -11,17 +11,10 @@ env:
   global:
     - BUNDLE_JOBS=4
     - NOKOGIRI_USE_SYSTEM_LIBRARIES=true
-matrix:
-  include:
-    - rvm: 1.9.3
-      gemfile: gemfiles/2.3.gemfile
-      env: QMAKE=/usr/lib/x86_64-linux-gnu/qt5/bin/qmake
-    - rvm: 1.9.3
-      gemfile: gemfiles/2.4.gemfile
-      env: QMAKE=/usr/lib/x86_64-linux-gnu/qt5/bin/qmake
+    - QMAKE=/usr/lib/x86_64-linux-gnu/qt5/bin/qmake
 gemfile:
   - gemfiles/2.3.gemfile
   - gemfiles/2.4.gemfile
 before_install:
-  - sh -c 'if [ -n "$QMAKE" ]; then sudo apt-add-repository -y ppa:ubuntu-sdk-team/ppa && sudo apt-get update && sudo apt-get install libqt5webkit5-dev qtdeclarative5-dev; fi'
+  - sh -c 'sudo apt-add-repository -y ppa:ubuntu-sdk-team/ppa && sudo apt-get update && sudo apt-get install libqt5webkit5-dev qtdeclarative5-dev'
```

Comment:
I think if we add `qt5-default` to this list, we should be able to omit the `QMAKE` environment variable on line 14.